include/kframework/evm.md

```diff
@@ -818,18 +805,16 @@ These are just used by the other operators for shuffling local execution state a
          </account>
       requires VALUE >Int ORIGFROM
 
-    rule <k> (. => #newAccount ACCTTO) ~> #transferFunds ACCTFROM ACCTTO VALUE ... </k>
-         <activeAccounts> ACCTS </activeAccounts>
+    rule <k> #transferFunds ACCTFROM ACCTTO VALUE => #transferFundsToNonExistent ACCTFROM ACCTTO VALUE ... </k> [owise]
+
+    rule <k> #transferFundsToNonExistent ACCTFROM ACCTTO VALUE => #newAccount ACCTTO ~> #transferFunds ACCTFROM ACCTTO VALUE ... </k>
```

Comment:
```suggestion
    rule <k> (. => #newAccount ACCTTO) ~> #transferFunds ACCTFROM ACCTTO VALUE ... </k> [owise]
```

Instead of making the new `#transferFundsNonExistent`, doesn't using this pattern work?

Reply:
Not really, `#transferFundsNonExistent` is used to differentiate the cases when `Gemptyisnonexistent <<SCHED >>` is true or not.

Reply:
Ok, sounds good. But looking at the rules, it seems that the case where `ACCTTO ==K ACCTFROM`, _and_ the account does _not_ exist in the configuration, is not handled. I guess this case doesn't usually make sense, but maybe it should be handled explicitly? Right now, it will just get stuck with `#transferFundsNonExistent` on the `<k>` cell.

Reply:
@ehildenb Well, the rules don't cover the case when the `ACCTFROM` is not in the configuration. A rule for this case would also include the scenario you mentioned.